packages/design-system/src/components/index.js

```diff
@@ -30,4 +30,9 @@ export { default as TabPanel } from './Tabs/TabPanel';
 export { default as TextField, unmaskValue } from './TextField/TextField';
 export { default as Tooltip } from './Tooltip/Tooltip';
 export { default as TooltipIcon } from './Tooltip/TooltipIcon';
+export { default as UsaBanner } from './UsaBanner/UsaBanner';
+export { default as UsaFlagIcon } from './UsaBanner/UsaFlagIcon';
```

Comment:
I don't think we need to export these icons, as they are only used inside the banner component. If they were generalized icons that we wanted folks to use I think it make sense to export them